### PR TITLE
nav_footer

### PR DESCRIPTION
**Pull request by Henrilang**

### CODE REVIEW

File: app/assets/stylesheets/components/_footer_nav.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.footer {
```
Comment (Shakey343) [inline app/assets/stylesheets/components/_footer_nav.scss:1]: .footer is a bootstrap class already - maybe call this something more specific like abc-footer?

File: app/assets/stylesheets/components/_footer_nav.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  background-color: #FF2840;
```
Comment (Shakey343) [inline app/assets/stylesheets/components/_footer_nav.scss:2]: Background-colors have been specified in the colors.scss file please make sure to use them here instead

File: app/views/shared/_footer_nav.html.erb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<div class="footer row">
```
Comment (Shakey343) [inline app/views/shared/_footer_nav.html.erb:1]: This 'footer' class will then need to change as well

**Review by Shakey343 — CHANGES_REQUESTED** [review]

File: app/views/shared/_footer_nav.html.erb (hunk 3 above)
Comment (Henrilang) [reply app/views/shared/_footer_nav.html.erb:1]: done

File: app/assets/stylesheets/components/_footer_nav.scss (hunk 1 above)
Comment (Henrilang) [reply app/assets/stylesheets/components/_footer_nav.scss:1]: done

File: app/assets/stylesheets/components/_footer_nav.scss (hunk 2 above)
Comment (Henrilang) [reply app/assets/stylesheets/components/_footer_nav.scss:2]: done

**Pull request merged**